too many bytes to write to stream error

When the server response the request, it is giving an error:

skript:

```
command /webbaşlat:
  trigger:
    start http server in port 25000 with context "/test/"

on http request received:
    set {_request} to event-httpexchange
    set {_method} to http request method of {_request}
    set {_body} to http request body of {_request}
    set {_userAgent} to http request {_request} request header value from key "User-Agent"
    set {_path} to http request context path of {_request}
    broadcast "HTTP REQUEST!"
    broadcast "METHOD: %{_method}%"
    broadcast "BDOY: %{_body}%"
    broadcast "UA: %{_userAgent}%"
    broadcast "Context: %{_path}%"
    
    reply {_request} with body "%{web::ad1::value::*}%" and response code 2100
    exit
on web request response:
    set {_response} to event-httpresponse
    set {_body} to body of {_response}
    set {_method} to request method of {_response}
    broadcast " "
    broadcast "server: %{_response}'s header value from key "server"%"
    broadcast "1: %{_body}%"
    broadcast "2: %status code of {_response}%"
    broadcast "3: %request method of {_response}%"   
on kci container action "ad1":
  if {web::ad1::%player%} is 1:
    set {web::ad1::%player%} to 1
    set {_randominteger} to random integer between 10000 and 999999999999999999
    delete {web::ad1::value::%player%}
    set {web::ad1::value::%player%} to "Odulunuzu Almak Icın Bu Komudu Kullanin: /reklamkodugir %player% %{_randominteger}%"
    send "<url:https://bit.ly/MNAD-1>&a&l[Reklam İçin Tıkla!]<reset>" to player

  if {web::ad1::%player%} is not set:
    set {web::ad1::%player%} to 1
    send "<url:https://bit.ly/MNAD-1>&a&l[Reklam İçin Tıkla!]<reset>" to player   
command /reklamkodugir [<text>] [<text>]:
  trigger:
    if {web::ad1::%player%} is 1:
      if arg-1 is "%player%":
        if arg-2 is {web::ad1::value::%player%}:
          
          set {web::ad1::%player%} to 0 
    send {web::ad1::%player%} to player
    send {web::ad1::value::%player%} to player          

```

error log:


```
[18:52:13 INFO]: MaliWhiteTea issued server command: /sk reload web.sk
[18:52:19 INFO]: HTTP REQUEST!
[18:52:19 INFO]: METHOD: GET
[18:52:19 INFO]: BDOY:
[18:52:19 INFO]: UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36 OPR/107.0.0.0 (Edition std-1)
[18:52:19 INFO]: Context: /test/
[18:52:19 INFO]: [Skript]     An error occurred while send response to request!
[18:52:19 ERROR]: #!#!
[18:52:19 ERROR]: #!#! [Skript] Severe Error:
[18:52:19 ERROR]: #!#!
[18:52:19 ERROR]: #!#! You're running outdated version of Skript! Please try updating it NOW; it might fix this.
[18:52:19 ERROR]: #!#! Run /sk update check to get a download link to latest Skript!
[18:52:19 ERROR]: #!#! You will be given instructions how to report this error if it persists after update.
[18:52:19 ERROR]: #!#!
[18:52:19 ERROR]: #!#! Stack trace:
[18:52:19 ERROR]: #!#! java.lang.RuntimeException: java.io.IOException: too many bytes to write to stream
[18:52:19 ERROR]: #!#!     at SkriptWebAPI-0.1.1.jar//dev.f2a.addon.skriptwebapi.elements.server.effects.EffSendHttpResponse.execute(EffSendHttpResponse.java:62)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.lang.Effect.run(Effect.java:49)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:88)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.lang.Trigger.execute(Trigger.java:52)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.SkriptEventHandler.lambda$check$3(SkriptEventHandler.java:155)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.SkriptEventHandler.lambda$check$4(SkriptEventHandler.java:166)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.util.Task.callSync(Task.java:164)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.util.Task.callSync(Task.java:148)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.SkriptEventHandler.check(SkriptEventHandler.java:164)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.SkriptEventHandler.access$000(SkriptEventHandler.java:53)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.SkriptEventHandler$PriorityListener.lambda$new$0(SkriptEventHandler.java:66)
[18:52:19 ERROR]: #!#!     at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:81)
[18:52:19 ERROR]: #!#!     at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[18:52:19 ERROR]: #!#!     at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54)
[18:52:19 ERROR]: #!#!     at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126)
[18:52:19 ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615)
[18:52:19 ERROR]: #!#!     at SkriptWebAPI-0.1.1.jar//dev.f2a.addon.skriptwebapi.internal.http.SkebHttpServer$SkebHttpHandler.lambda$handle$0(SkebHttpServer.java:52)
[18:52:19 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftFuture.run(CraftFuture.java:88)
[18:52:19 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:480)
[18:52:19 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1479)
[18:52:19 ERROR]: #!#!     at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:446)
[18:52:19 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1393)
[18:52:19 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1170)
[18:52:19 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317)
[18:52:19 ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:833)
[18:52:19 ERROR]: #!#! Caused by: java.io.IOException: too many bytes to write to stream
[18:52:19 ERROR]: #!#!     at jdk.httpserver/sun.net.httpserver.FixedLengthOutputStream.write(FixedLengthOutputStream.java:79)
[18:52:19 ERROR]: #!#!     at java.base/java.io.FilterOutputStream.write(FilterOutputStream.java:108)
[18:52:19 ERROR]: #!#!     at jdk.httpserver/sun.net.httpserver.PlaceholderOutputStream.write(ExchangeImpl.java:459)
[18:52:19 ERROR]: #!#!     at SkriptWebAPI-0.1.1.jar//dev.f2a.addon.skriptwebapi.elements.server.effects.EffSendHttpResponse.execute(EffSendHttpResponse.java:58)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.lang.Effect.run(Effect.java:49)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:88)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.lang.Trigger.execute(Trigger.java:52)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.SkriptEventHandler.lambda$check$3(SkriptEventHandler.java:155)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.SkriptEventHandler.lambda$check$4(SkriptEventHandler.java:166)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.util.Task.callSync(Task.java:164)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.util.Task.callSync(Task.java:148)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.SkriptEventHandler.check(SkriptEventHandler.java:164)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.SkriptEventHandler.access$000(SkriptEventHandler.java:53)
[18:52:19 ERROR]: #!#!     at Skript2.8.0.jar//ch.njol.skript.SkriptEventHandler$PriorityListener.lambda$new$0(SkriptEventHandler.java:66)
[18:52:19 ERROR]: #!#!     at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:81)
[18:52:19 ERROR]: #!#!     at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[18:52:19 ERROR]: #!#!     at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54)
[18:52:19 ERROR]: #!#!     at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126)
[18:52:19 ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615)
[18:52:19 ERROR]: #!#!     at SkriptWebAPI-0.1.1.jar//dev.f2a.addon.skriptwebapi.internal.http.SkebHttpServer$SkebHttpHandler.lambda$handle$0(SkebHttpServer.java:52)
[18:52:19 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftFuture.run(CraftFuture.java:88)
[18:52:19 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:480)
[18:52:19 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1479)
[18:52:19 ERROR]: #!#!     at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:446)
[18:52:19 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1393)
[18:52:19 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1170)
[18:52:19 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317)
[18:52:19 ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:833)
[18:52:19 ERROR]: #!#!
[18:52:19 ERROR]: #!#! Version Information:
[18:52:19 ERROR]: #!#!   Skript: 2.8.0 (OUTDATED)
[18:52:19 ERROR]: #!#!     Flavor: skriptlang-github
[18:52:19 ERROR]: #!#!     Date: 18:20:11.223179900
[18:52:19 ERROR]: #!#!   Bukkit: 1.20.1-R0.1-SNAPSHOT
[18:52:19 ERROR]: #!#!   Minecraft: 1.20.1
[18:52:19 ERROR]: #!#!   Java: 18.0.2.1 (Java HotSpot(TM) 64-Bit Server VM 18.0.2.1+1-1)
[18:52:19 ERROR]: #!#!   OS: Windows Server 2019 amd64 10.0
[18:52:19 ERROR]: #!#!
[18:52:19 ERROR]: #!#! Server platform: Paper
[18:52:19 ERROR]: #!#!
[18:52:19 ERROR]: #!#! Current node: null
[18:52:19 ERROR]: #!#! Current item:
[18:52:19 ERROR]: #!#! Current trigger: http request received () (web.sk, line 5)
[18:52:19 ERROR]: #!#!
[18:52:19 ERROR]: #!#! Thread: Server thread
[18:52:19 ERROR]: #!#!
[18:52:19 ERROR]: #!#! Language: turkish
[18:52:19 ERROR]: #!#! Link parse mode: DISABLED
[18:52:19 ERROR]: #!#!
[18:52:19 ERROR]: #!#! End of Error.
[18:52:19 ERROR]: #!#!
[18:53:51 WARN]: Can't keep up! Is the server overloaded? Running 5274ms or 105 ticks behind
```

--- a/src/main/java/dev/f2a/addon/skriptwebapi/elements/server/effects/EffSendHttpResponse.java
+++ b/src/main/java/dev/f2a/addon/skriptwebapi/elements/server/effects/EffSendHttpResponse.java
@@ -54,7 +54,8 @@ public class EffSendHttpResponse extends Effect {
         }
 
         try {
-            exchange.sendResponseHeaders(code, body.length());
+            byte[] bytes = body.getBytes(StandardCharsets.UTF_8);
+            exchange.sendResponseHeaders(code, bytes.length);
             exchange.getResponseBody().write(body.getBytes(StandardCharsets.UTF_8));
             exchange.close();
         } catch (IOException ex) {

